### PR TITLE
grc: Accept failures in message sending

### PR DESCRIPTION
**Pull request by mbr0wn**
This fixes a rare corner case seen in unit tests, where multiple tests interfere with one another.

During the execution of unit tests for the pluggable workflow PR, this sometimes happened. I couldn't quite track it down, but when you run multiple GRC-related tasks in quick succession in a unit test, the output messenger would fail to open-and-close the file descriptor.

## Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/gnuradio/gnuradio/blob/main/CONTRIBUTING.md).
- [x] I have squashed my commits to have one significant change per commit. 
- [x] I [have signed my commits before making this PR](https://github.com/gnuradio/gnuradio/blob/main/CONTRIBUTING.md#dco-signed)
- [x] My code follows the code style of this project. See [GREP1.md](https://github.com/gnuradio/greps/blob/main/grep-0001-coding-guidelines.md).
- [x] I have updated [the documentation](https://wiki.gnuradio.org/index.php/Main_Page#Documentation) where necessary.
- [x] I have added tests to cover my changes, and all previous tests pass.

### CODE REVIEW

File: grc/core/Messages.py [hunk 1]
```diff
@@ -40,7 +40,10 @@ def send(message):
         message: a message string
     """
     for messenger in MESSENGERS_LIST:
-        messenger(_indent + message)
+        try:
+            messenger(_indent + message)
+        except:
+            pass
```
Comment (marcusmueller) [inline grc/core/Messages.py:46]: I feel like we should be at least logging this? Otherwise, we'll have hard-to-debug problems if for any reason in the future, a crucial messenger permanently fails.

**Comment by marcusmueller** [comment]
Just for understanding: which messenger is affected here? stdout.write? Could you add that to your commit msg, please to avoid future confusion :)

File: grc/core/Messages.py (hunk 1 above)
Comment (mbr0wn) [reply grc/core/Messages.py:46]: You mean, using the regular Python logger instead of the messenger logger? Sure we can add that.

**Comment by mbr0wn** [comment]
> Just for understanding: which messenger is affected here? stdout.write? Could you add that to your commit msg, please to avoid future confusion :)

✔️

**Review by marcusmueller — APPROVED** [review]
